File: spoj/army.py
t=raw_input()
t=int(t)
while t:
	print
	n=raw_input()
	ngl=raw_input()
	ngl=ngl.split(" ")
	ngl=max([int(i) for i in ngl])
	nml=raw_input()
	nml=nml.split(" ")
	nml=max([int(i) for i in nml])
	if ngl>=nml:
		print "Godzilla"
	else:
		print "MechaGodzilla"
		
	t-=1

File: spoj/army.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
	{
		int t;
		cin>>t;
		while(t--)
		{
			cout<<endl;
			int ng,nw;
			cin>>ng>>nw;
			int ngl[ng],nwl[nw],maxg=0,maxw=0;
			for(int i=0;i<ng;i++)
			{
				cin>>ngl[i];
				if(ngl[i]>=maxg)
					maxg=ngl[i];
			}
			for(int i=0;i<nw;i++){
				cin>>nwl[i];
				if(nwl[i]>=maxw)
					maxw=nwl[i];
			}
			if(maxg>=maxw)
				cout<<"Godzilla"<<endl;
			else
				cout<<"MechaGodzilla"<<endl;
		
		}
		return 0;
	}